Testing/xASL_test_IntegrationTesting.m

```diff
@@ -132,7 +132,6 @@
     xASL_delete(fullfile(testingRoot,'FlavorDatabaseTest',flavorName,'rawdata'),1);
     
     % Create default dataPar.json
-    defaultDataPar.x.bUseMNIasDummyStructural = 1; % when no structural data, use ASL-MNI registration
```

Comment:
This parameter seems necessary for the flavor database, which hasn't got structural images for many flavors.

Reply:
You are of course right. My fault of insisting that Michael should remove it... I'm reverting this change.